app/client/src/components/pages/StateTribal.js

```diff
@@ -178,13 +178,24 @@ function StateTribal() {
   const services = useServicesContext();
   const tribeMapping = useTribeMappingContext();
 
+  const {
+    activeState,
+    errorType,
+    introText,
+    setActiveState,
+    setErrorCode,
+    setUsesStateSummaryServiceError,
+    usesStateSummaryServiceError,
+  } = useContext(StateTribalTabsContext);
+
   // redirect to '/stateandtribe' if the url is /state or /tribe
   useEffect(() => {
     const pathname = window.location.pathname.toLowerCase();
     if (['/state', '/state/', '/tribe', '/tribe/'].includes(pathname)) {
+      setErrorCode('');
       navigate('/state-and-tribal', { replace: true });
     }
-  }, [navigate]);
+  }, [navigate, setErrorCode]);
```

Comment:
Typo here, `setErrorCode` should be `setErrorType`.

Reply:
I'll go ahead and make the fix!

Reply:
Good catch! Thank you for fixing that as well!